docs/docs/configuration/object_detectors.md

```diff
@@ -682,6 +682,29 @@ model:
 
 Note that the labelmap uses a subset of the complete COCO label set that has only 80 objects.
 
+#### YOLOx
+
+[YOLOx](https://github.com/Megvii-BaseDetection/YOLOX) models are supported, but not included by default. See [the models section](#downloading-yolo-models) for more information on downloading the YOLOx model for use in Frigate.
+
+After placing the downloaded onnx model in your config folder, you can use the following configuration:
+
+```yaml
+detectors:
+  onnx:
+    type: onnx
+
+model:
+  model_type: yolox
+  width: 416 # <--- should match the imgsize set during model export
+  height: 416 # <--- should match the imgsize set during model export
+  input_tensor: nchw_denorm
```

Comment:
Heads up: Frigate crashes on this option when I try to run `yolox` model with these instructions with `ghcr.io/blakeblackshear/frigate:68382d8-rocm` image: 

```js
2025-04-18 20:56:07.624418624  Line #  : 74
2025-04-18 20:56:07.624419914  Key     : model -> input_tensor
2025-04-18 20:56:07.624420494  Value   : nchw_denorm
2025-04-18 20:56:07.624427754  Message : Input should be 'nchw', 'nhwc', 'hwnc' or 'hwcn'
```

Reply:
the docs are wrong, it should be `float_denorm`